docs/weave-tutorial/04-models.md

```diff
@@ -6,15 +6,14 @@ sidebar_label: Models
 
 > [PR#6](https://github.com/iov-one/tutorial/pull/6): _Create models_
 
-We defined our state in [codec section](weave-tutorial/03-codec.md). To use models in Weave we have to wrap our model with some functionalities and enforce it as a **morm.Model**.
-
-Ensure our `OrderBook` fulfills morm.Model:
+We defined our state in [codec section](weave-tutorial/03-codec.md). Models are wrapped codec with functionalities to integrate codec with Weave.
+First ensure our `User` fulfills morm.Model:
 
 ```go
-var _ morm.Model = (*OrderBook)(nil)
+var _ morm.Model = (*User)(nil)
 ```
 
-This is just a helper that enforces OrderBook model to fulfill morm.Model interface so that if you forget to implement a method compiler will complain. Guaranteeing it _I am trying to implement this interface_.
+This is just a helper that enforces User model to fulfill morm.Model interface so that if you forget to implement a method compiler will complain. Guaranteeing it _I am trying to implement this interface_.
```

Comment:
```suggestion
This is just a helper that enforces User model to fulfill morm.Model interface so that if you forget to implement a method, the compiler will complain. Guaranteeing it _I am trying to implement this interface_.
```